src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -109,6 +109,7 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
         const client: PostgreSQLManagementClient = createAzureClient(this.root, PostgreSQLManagementClient);
         const deletingMessage: string = `Deleting server "${this.name}"...`;
         await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification, title: deletingMessage }, async () => {
+            await this.deletePostgresCredentials();
```

Comment:
Mmm I feel like this should be after `deleteMethod`? Like if the delete fails, we probably don't need to delete the credentials